ui/i18n/src/main/res/values/strings.xml

```diff
@@ -572,6 +572,8 @@
     <string name="pref_new_episodes_action_title">New episodes action</string>
     <string name="pref_new_episodes_action_sum">Action to take for new episodes</string>
     <string name="episode_information">Episode information</string>
+    <string name="pref_global_default_episode_list_sort_order_title">Default Episode Order</string>
```

Comment:
We switched to lower case titles recently (except for the first letter). Also, I would mention "sort" because it's called "sort" everywhere else in the app.

```suggestion
    <string name="pref_global_default_episode_list_sort_order_title">Default sort order</string>
```